### DISCUSSION TITLE
Download an image from a private repo fails with `This private-user-images.githubusercontent.com page can’t be found`

### DISCUSSION THREAD

**Discussion by AlvaroStream** [community/community#79214] [Other Feature Feedback, Questions, & Ideas]
### Select Topic Area

Product Feedback

### Body

When there is an image in a GH issue that I can see in the discussion on a private repo, I can't download the image or zoom in, which is annoying. Sometimes it happens even with images I have uploaded.

Is there any way to make it work.

I'm using Chrome `Version 120.0.6099.109 (Official Build) (arm64)` in MAC OS

**Comment by shenweiyan** [comment]
I've encountered a similar issue. It would be great if there was a way to download images from private repository issues or discussions.

**Comment by RkShaRkz** [comment]
Can confirm it's still happening. Example issue that reproes this quite clearly: https://github.com/RkShaRkz/ExoticaTechnologies/issues/10

Why is this still happening while I remember it clearly not being a problem on my work account?
